src/registrar/fixtures.py

```diff
@@ -268,10 +268,11 @@ class DomainApplicationFixture:
             "status": "action needed",
             "organization_name": "Example - Action Needed",
         },
-        {
-            "status": "rejected",
-            "organization_name": "Example - Rejected",
-        },
+        # Pa11y does not like this fixture
```

Comment:
I wonder if storing this in a variable, or parts of it, would make a difference. Its weird that it doesn't like this when its text content follows a similar pattern to everything else. The only other thing I can think of might be that it doesn't like the length of this dictionary?

Reply:
Nope on length. I tried removing Action Needed and keeping Rejected, still freaks Pa11y out.